style.css

```diff
@@ -127,26 +154,7 @@ body {
   flex: none;
   order: 2;
   flex-grow: 0;
-}
-
-.social-media {
-  margin-left: 2em;
-  margin-right: 24 px;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  padding: 0;
-  width: 13em;
-  flex: none;
-  order: 3;
-  flex-grow: 0;
-  text-decoration: none;
-}
-
-.social-media > li {
-  list-style: none;
-  text-decoration: none;
+  max-width: 50vh;
 }
 
 .works {
```

Comment:
Great job on making some of the previous reviewers requested changes your cards have hover effect now 👍💪, but we still need to fix some small padding and margin issues in get it perfect , your cards needs a padding bottom as in figma design and the margin between the first demo card and the list of cards as pointed in this image 

**your design : **
![cards padding and margin student](https://user-images.githubusercontent.com/59707859/229284849-c29b109c-bca9-4cf2-9141-e132ba3136a5.PNG)
 
**figma template : **
![cards padding and margin](https://user-images.githubusercontent.com/59707859/229284872-ef1cc147-fc0d-496a-b7cd-dbe66f6460bd.PNG)

Reply:
It's Done